.storybook/.babelrc.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const path = require('path');
+const packageJson = require('../package.json');
+
+const root = path.resolve(__dirname, '../');
+const babelConfig = Object.keys(packageJson.babel).reduce((acc, key) => {
+  const options = packageJson.babel[key].map(option => {
```

Comment:
we're assuming that `option` is always a string right? do we need to account for plugins like https://github.com/IBM/carbon-components-react/blob/master/package.json#L250-L255?